app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 in_class: int,
+                 in_mark: int,
+                 in_brand: str) -> None:
+        self.comfort_class = in_class
+        self.clean_mark = in_mark
+        self.brand = in_brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 in_distance: float,
+                 in_power: int,
+                 in_rating: float,
+                 in_count: int) -> None:
+        self.distance_from_city_center = in_distance
+        self.clean_power = in_power
+        self.average_rating = in_rating
+        self.count_of_ratings = in_count
+
+    def calculate_washing_price(self, in_car: Car) -> float:
+        price = in_car.comfort_class                    \
+            * (self.clean_power - in_car.clean_mark)    \
+            * self.average_rating                       \
+            / self.distance_from_city_center
```

Comment:
According to the checklist to this task presence of \ sign is bad practice, please fix it

Reply:
ok